scripts/poppunk_batch_mst.py

```diff
@@ -0,0 +1,307 @@
+    # analysis options
+    aGroup = parser.add_argument_group('Analysis options')
+    aGroup.add_argument('--rank', help='Comma separated ranks used to fit lineage model (list of ints)',
+                                                type = str,
+                                                default = "10")
+    aGroup.add_argument('--threads', help='Number of threads for parallelisation (int)',
+                                                type = int,
+                                                default = 1)
+    aGroup.add_argument('--use-gpu', help='Use GPU for analysis',
+                                                default=False,
+                                                action='store_true')
+    aGroup.add_argument('--deviceid', help='GPU device ID (int)',
+                                                type = int,
+                                                default = 0)
+    aGroup.add_argument('--db-args', help="Other arguments to pass to poppunk. e.g. "
+                                             "'--min-k 13 --max-k 29'",
+                                                default = "")
+    aGroup.add_argument('--model-args', help="Other arguments to pass to lineage model fit",
+                                                default = "")
+    aGroup.add_argument('--assign-args', help="Other arguments to pass to poppunk_assign",
+                                                default = "")
+
+    # Executable options
+    eGroup = parser.add_argument_group('Executable locations')
+    eGroup.add_argument('--poppunk-exe', help="Location of poppunk executable. Use "
+                                             "'python poppunk-runner.py' to run from source tree",
+                                                default="poppunk")
+    eGroup.add_argument('--assign-exe', help="Location of poppunk_assign executable. Use "
+                                             "'python poppunk_assign-runner.py' to run from source tree",
+                                                default="poppunk_assign")
+    eGroup.add_argument('--mst-exe', help="Location of poppunk executable. Use "
+                                           "'python poppunk_mst-runner.py' to run from source tree",
+                                                default="poppunk_mst")
+
+    return parser.parse_args()
+
+def writeBatch(rlines, batches, batch_selected, use_names = False):
+    tmpdir = ""
+    if use_names:
+        tmpdir = "./pp_mst_" + batch_selected
+        os.mkdir(tmpdir)
+    else:
+        tmpdir = tempfile.mkdtemp(prefix="pp_mst", dir="./")
+    with open(tmpdir + "/" + rfile_names, 'w') as outfile:
+        for rline, batch in zip(rlines, batches):
+            if batch == batch_selected:
+                outfile.write(rline)
+
+    return tmpdir
+
+def runCmd(cmd_string):
+    sys.stderr.write("Running command:\n")
+    sys.stderr.write(cmd_string)
```

Comment:
Missing newline (my fault)
```suggestion
    sys.stderr.write(cmd_string + "\n")
```

Reply:
Another missing new line fixed in f823964.